internal/exec/stages/mount/mount.go

```diff
@@ -41,6 +41,8 @@ const (
 	name = "mount"
 )
 
+var pathExists = false
```

Comment:
Since this is a global variable, if it's set once, it'll affect every subsequent `mountFs()` call, which is certainly not what we want.

Reply:
makes perfect sense didn't realise that at the time thanks!